firebase_admin/_messaging_utils.py

```diff
@@ -161,11 +90,67 @@ class AndroidNotification(object):
             in ``title_loc_key`` (optional).
         channel_id: channel_id of the notification (optional).
         image: Image url of the notification (optional).
+        ticker: Sets the ``ticker`` text, which is sent to accessibility services. Prior to API
+            level 21 (Lollipop), sets the text that is displayed in the status bar when the
+            notification first arrives (optional).
+        sticky: When set to ``false`` or unset, the notification is automatically dismissed when the
+            user clicks it in the panel. When set to ``True``, the notification persists even when
+            the user clicks it (optional).
+        event_timestamp: For notifications that inform users about events with an absolute time
+            reference, sets the time that the event in the notification occurred. Notifications
+            in the panel are sorted by this time (optional).
+        local_only: Set whether or not this notification is relevant only to the current device.
+            Some notifications can be bridged to other devices for remote display, such as a Wear OS
+            watch. This hint can be set to recommend this notification not be bridged (optional).
+            See Wear OS guides:
+            https://developer.android.com/training/wearables/notifications/bridger#existing-method-of-preventing-bridging
+        priority: Set the relative priority for this notification. Low-priority notifications may be
+            hidden from the user in certain situations. Note this priority differs from
+            ``AndroidMessagePriority``. This priority is processed by the client after the message
+            has been delivered. Whereas ``AndroidMessagePriority`` is an FCM concept that controls
+            when the message is delivered (optional). Must be one of ``default``, ``min``, ``low``,
+            ``high``, ``max`` or ``normal``.
+        vibrate_timings_millis: Set the vibration pattern to use. Pass in an array of seconds
```

Comment:
Description says `array of seconds` while the argument name says `millis`. Please clarify.

Reply:
Fixed. Thanks!!